ci-git-release.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+if [ "$CIRCLE_BRANCH" = "jqian/detectsdk" ]; then
```

Comment:
Intent to have "jqian/detectsdk"?

Reply:
I no longer use this script... as all the logic is now part of the circleci config.yml... I will delete the script file